src/utils/number.ts

```diff
@@ -7,7 +7,7 @@
  * @param number The number to format.
  * @returns A string representing the formatted number.
  */
-export const formatNumberCompact = (number: number): string => {
+export const Number = (number: number): string => {
```

Comment:
function naming should more clear, and check if `Number` is a keyword.

Reply:
Fixed